testsuite/python/tabulated_sympy.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2013-2022 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import unittest as ut
+import unittest_decorators as utx
+import espressomd
+import espressomd.interactions
+import numpy as np
+from sympy import symbols, sympify
+
+class TabulatedTest(ut.TestCase):
```

Comment:
sympy should be available in some of the CI containers. You can disable `TabulatedTest` when sympy is missing with fixture `@utx.skipIfMissingModules("sympy")` above the class name. Don't forget to add a linter rule exception for unused imports.